File: 04 Kontrollstrukturen/Pascal/Aufgabe10.pas
Program Aufgabe10;

{$codepage utf8}

Var Auswahl: Integer;

Begin
    Repeat
        WriteLn();
        WriteLn();
        WriteLn();

        WriteLn('Weißes Haus Velten - Menü');
        WriteLn('1 - Döner Teller (groß)');
        WriteLn('2 - Döner Teller (normal)');
        WriteLn('3 - Kids Menü');
        WriteLn('4 - Chili Cheeseburger Menü');
        WriteLn('5 - Chili Hamburger Menü');
        WriteLn('6 - Family Menü');
        WriteLn('7 - Beenden');

        WriteLn();

        Write('Bitte triff eine Auswahl: ');

        Read(Auswahl);

        Case Auswahl Of
            1: WriteLn('Döner Teller (groß) mit doppelt Fleisch, Pommes frites und Salat: 8,50€');
            2: WriteLn('Döner Teller (normal) mit Pommes frites und Salat: 7,80€');
            3: WriteLn('Kids Menü - 4 Nuggets - Pommes frites - Capri-Sonne - Überraschung: 5,00€');
            4: WriteLn('Chili Cheeseburger Menü - 1 Chili Cheeseburger - 1 Pommes frites - 1 Softgetränk 0,33l: 6,80€');
            5: WriteLn('Chili Hamburger Menü - 1 Chili Hamburger - 1 Pommes frites - 1 Softgetränk 0,33l: 6,50€');
            6: Begin
                   WriteLn('Family Menü - 1 Dönerteller mit Salat und Pommes frites - 2 Hamburger');
                   WriteLn(' - Chicken Nuggets (6 Stück) - 1 Pommes frites - 2 Softgetränke 0,33l - 2 Capri-Sonnen: 28,50€');
               End;
            7: WriteLn('Bye!');
        End;
    Until Auswahl = 7;
End.

File: 04 Kontrollstrukturen/Go/Aufgabe10.go
package main;

import "fmt"

var auswahl int

func main() {
    for {
        fmt.Println();
        fmt.Println();
        fmt.Println();

        fmt.Println("Weißes Haus Velten - Menü");
        fmt.Println("1 - Döner Teller (groß)");
        fmt.Println("2 - Döner Teller (normal)");
        fmt.Println("3 - Kids Menü");
        fmt.Println("4 - Chili Cheeseburger Menü");
        fmt.Println("5 - Chili Hamburger Menü");
        fmt.Println("6 - Family Menü");
        fmt.Println("7 - Beenden");

        fmt.Println();

        fmt.Print("Bitte triff eine Auswahl: ");

        fmt.Scan(&auswahl);

        switch auswahl {
            case 1:
				fmt.Println("Döner Teller (groß) mit doppelt Fleisch, Pommes frites und Salat: 8,50€");
            case 2:
				fmt.Println("Döner Teller (normal) mit Pommes frites und Salat: 7,80€");
            case 3:
				fmt.Println("Kids Menü - 4 Nuggets - Pommes frites - Capri-Sonne - Überraschung: 5,00€");
            case 4:
				fmt.Println("Chili Cheeseburger Menü - 1 Chili Cheeseburger - 1 Pommes frites - 1 Softgetränk 0,33l: 6,80€");
            case 5:
				fmt.Println("Chili Hamburger Menü - 1 Chili Hamburger - 1 Pommes frites - 1 Softgetränk 0,33l: 6,50€");
            case 6:
				fmt.Println("Family Menü - 1 Dönerteller mit Salat und Pommes frites - 2 Hamburger");
				fmt.Println(" - Chicken Nuggets (6 Stück) - 1 Pommes frites - 2 Softgetränke 0,33l - 2 Capri-Sonnen: 28,50€");
			case 7:
				fmt.Println("Bye!");
		}

		if auswahl == 7 {
			break
		}
	}
}